pkg/providers/ovirt/client/ovirt-client.go

```diff
@@ -0,0 +1,270 @@
+package ovirtclient
+
+import (
+	"fmt"
+
+	ovirtsdk "github.com/ovirt/go-ovirt"
+)
+
+// OvirtClient retrieves rich VM object - with all necessary links have been followed
+type OvirtClient interface {
+	GetVM(connectionSettings *ConnectionSettings, id *string, name *string, cluster *string) (*ovirtsdk.Vm, error)
+}
+
+// ConnectionSettings wrap information required to make oVirt API connection
+type ConnectionSettings struct {
+	URL      string
+	Username string
+	Password string
+	Insecure bool
```

Comment:
We need a field for CA content to make the connection secure.

Reply:
:+1: